File: backend/javaserver/src/main/java/de/hsrm/swt02/model/MetaState.java
package de.hsrm.swt02.model;

/**
 * Enum for the state of a Step, meant to be used in the list MetaEntry.
 */
public enum MetaState {

    // values
    INACTIVE, OPEN, BUSY, DONE;

    /**
     * value getter.
     * @return name()
     */
    public String value() {
        return name();
    }

    /**
     * valueof getter.
     * @param v is string corresponding the enum value
     * @return int
     */
    public static MetaState fromValue(String v) {
        return valueOf(v);
    }

}

File: frontend/CommunicationLib/Model/DataModel/MetaState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommunicationLib.Model
{
    /// <summary>
    /// This Enum represents the state of a Step in the MetaEntryList
    /// </summary>
    public enum MetaState
    {
        INACTIVE,
        OPEN,
        BUSY,
        DONE,
    }
}
